app/src/main/java/com/github/se/wanderpals/ui/screens/trip/map/Map.kt

```diff
@@ -284,10 +286,22 @@ fun Map(
                 state = markerState,
                 title = "Click to Create Suggestions",
                 onInfoWindowClick = {
```

Comment:
Move setVariablesSuggestionId and Location to setVariablesSuggestion since they are always set when setVariablesSuggestion is called

Reply:
Nice!